### ISSUE TITLE
Time stamp too precise for validator

### ISSUE THREAD

**Issue by mih** [bids-standard/legacy-validator#1314]
With 1.5.4 of the validator I see:

```
1: [ERR] Entries in the "acq_time" column of _scans.tsv should be expressed in the following format YYYY-MM-DDTHH:mm:ss (year, month, day, hour (24h), minute, second; this is equivalent to the RFC3339 “date-time” format.  (code: 84 - ACQTIME_FMT)                                                                                                                   
                ./sub-03902/sub-03902_scans.tsv
                        @ line: 2
                        Evidence: 2014-09-30T13:18:14.235000
```

I do not think this is actually an error. The timestamp has the right format, it is just more precise, but still RFC3339 or rather ISO8601 compliant.

If the precision is intentionally limited, this could be a warning, as any standard datetime library should be able to parse this properly.

Apologies if this has been fixed in a later version already.

**Comment by sappelhoff** [comment]
Thanks for the report.

bids-validator 1.5.4 was released July 7th, 2020: https://github.com/bids-standard/bids-validator/tree/v1.5.4

I think your issue has meanwhile been fixed in newer versions since #1019 

Could you verify that, please?

**Comment by mih** [comment]
Ah sorry, I closed this minutes later, but github apparently did not let it through. Here was my message:

> Tested with 1.7.3, already fixed. Thx! And sorry for the noise.
